src/graph/backend/dnnl/passes/utils.cpp

```diff
@@ -646,6 +647,30 @@ bool inverse_mul_scales(std::shared_ptr<op_t> &scale_op) {
     return true;
 }
 
+bool check_broadcast_requirement_for_inputs_pair(
+        const std::shared_ptr<op_t> &op, size_t index1, size_t index2) {
```

Comment:
I remember i had a suggestion for this function signature.
```suggestion
bool need_broadcast_for_inputs(
        const std::shared_ptr<op_t> &op, size_t in1, size_t in2) {
```

Reply:
Fixed